resources/js/i18n/messages/en/auth.js

```diff
@@ -1,9 +1,11 @@
 export default {
   login: 'Login',
   register: 'Register',
+  alreadyregistered: 'Already registered?',
```

Comment:
please use camel casing for the translation keys. this one should be `alreadyRegistered`. the same goes for `passwordConfirmation`